pkg/controller/plan/validation.go

```diff
@@ -309,7 +309,7 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 		id := path.Join(
 			plan.TargetNamespace(),
 			ref.Name)
-		pErr = inventory.Get(&ocp.VM{}, id)
+		_, pErr = inventory.VM(&refapi.Ref{Name: id})
```

Comment:
inventory.Get() can no longer be used to resolve name references because the 2nd parameter (for OCP) is now a UID. This needs to be a _ref_ lookup just like vSphere.